packages/protocol-sdk/README.md

```diff
@@ -33,18 +33,49 @@ async function mintNFT(
 }
 ```
 
+### Creating an 1155 contract:
+
+If an object with {name, uri} is passed in to this helper, it uses the creatorAccount and those values to either 1) create or 2) mint to that existing contract.
+
+If you wish to mint on an existing contract, pass that contract in the contract field. The return value is the prepared transaction that you can use viem or wagmi to send.
+
+```ts
+import type { PublicClient } from "viem";
+import { create1155CreatorClient } from "@zoralabs/protocol-sdk";
+
+export async function createContract({
+  publicClient,
+  walletClient,
+}: {
+  publicClient: PublicClient;
+  walletClient: WalletClient;
+}) {
+  const creatorClient = create1155CreatorClient({ publicClient });
+  const { request } = await creatorClient.createNew1155Token({
+    contract: {
+      name: "testContract",
+      uri: demoContractMetadataURI,
+    },
+    tokenMetadataURI: demoTokenMetadataURI,
+    account: creatorAccount,
+    mintToCreatorCount: 1,
+  });
+  const { request: simulateRequest } = publicClient.simulateContract(request);
```

Comment:
thoughts on having it return this simulation result instead of submitting in one go?